tests/framework/OutStreams/tests

```diff
@@ -87,4 +87,17 @@
     input = 'sample_plot.xml'
     output = 'SamplePlot/plotter.png'
   [../]
+  
+  [./PopulationPlot]
+    type = 'RavenFramework'
+    input = 'ga_population_plot.xml'
+    output = 'GApopulation/plotter.png'
```

Comment:
here only check the generation of the file, not actually check the diff

Reply:
Fixed